devops/scripts/install_drivers.sh

```diff
@@ -108,33 +108,38 @@ InstallIGFX () {
   echo "Compute Runtime version $CR_TAG"
   echo "CM compiler version $CM_TAG"
   echo "Level Zero version $L0_TAG"
-  IS_IGC_DEV=$(CheckIGCdevTag $IGCTAG)
-  IGNORE_CHECKSUM=false
-  DPKG_OPTIONS=""
-  if [ "$IS_IGC_DEV" == "Yes" ]; then
-    echo "IGC dev git hash $IGC_DEV_VER"
-    get_pre_release_igfx $IGC_DEV_URL $IGC_DEV_VER
-    IGNORE_CHECKSUM=true
-    DPKG_OPTIONS=" --force-depends-version"
-  else
-    echo "IGC version $IGC_TAG"
-    get_release intel/intel-graphics-compiler $IGC_TAG \
-      | grep ".*deb" \
-      | wget -qi -
-  fi
+  echo "IGC version $IGC_TAG"
+  get_release intel/intel-graphics-compiler $IGC_TAG \
```

Comment:
Please add a comment why we do it unconditionally.

Reply:
Sure, this is to get rid of the *dependency* issue -- by install the igc first, we will satisfy all the dpkg dependencies .
Then when we install dev igc later, it will then be treated as *downgrade*, so that we don't need to use the risky *force-dependence-version*  option in dpkg.

Of course, this also installed the libopencl-clang so that we can copy and use later.

Reply:
I meant in the file itself.

Reply:
yes, understood, doing it.